### PR TITLE
create runnable jar

### PR DESCRIPTION
**Pull request by Usha3012**

### CODE REVIEW

**Comment by Usha3012** [comment]
For Controller , I need Entity .So, it will be good to do this part as part of issue 3.

**Comment by njoysubho** [comment]
you can return simple string now then in next PR change it

File: src/main/java/org/api/controller/CategoryController.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class CategoryController {
+   @RequestMapping(path = "/goals",method = RequestMethod.GET,produces = MediaType.APPLICATION_JSON_VALUE)
```
Comment (njoysubho) [inline src/main/java/org/api/controller/CategoryController.java]: it is a CategoryController why path is /goals 🤦‍♂ should be /categories

**Review by njoysubho — CHANGES_REQUESTED** [review]
see comments

File: src/main/java/org/api/controller/CategoryController.java (hunk 1 above)
Comment (Usha3012) [reply src/main/java/org/api/controller/CategoryController.java]: Changed :)

**Review by njoysubho — APPROVED** [review]

**Pull request merged**